harness/features/variable.cloud.test.ts

```diff
@@ -50,6 +51,16 @@ describe('Variable Tests - Cloud', () => {
             ),
         )
     }
+    function getEvalReason(
+        sdkName: string,
+        reason: string,
+        details?: string,
+        target_id?: string,
+    ) {
+        return sdkName === 'OF-NodeJS'
+            ? { reason }
```

Comment:
we are looking to potentially add support for the other properties in Android OF Provider, but not a required thing for this PR: https://github.com/DevCycleHQ/android-client-sdk/pull/227#discussion_r2216603453